gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,3 +1,4 @@
+#Wed Jul 15 17:15:38 IST 2020
```

Comment:
please remove the redundant change.

Reply:
Removed

Reply:
should it default to the pravega version?
For above comment, this should be individual parameter as its not dependent on pravega now. So I kept it as a new parameter.